cmd/skaffold/app/cmd/deploy.go

```diff
@@ -88,6 +89,21 @@ func getArtifactsToDeploy(out io.Writer, fromFile, fromCLI []build.Artifact, art
 	deployed = build.MergeWithPreviousBuilds(fromCLI, deployed)
 	deployed = build.MergeWithPreviousBuilds(fromFile, deployed)
 
+	if opts.CustomTag != "" {
```

Comment:
Can you leave a comment before this section just to clarify that this piece of code is applying the tags? Or move it into a function like `applyCustomTag(deployed)` whose name will clarify